<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_TestTableWidget</class>
 <widget class="QWidget" name="DB_TestTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblId">
          <property name="text">
           <string>Id</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leId"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblParent">
          <property name="text">
           <string>Parent</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leParent"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leName"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblValue1">
          <property name="text">
           <string>Value 1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="leValue1"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblValue2">
          <property name="text">
           <string>Value 2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="leValue2"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lblComboBox1">
          <property name="text">
           <string>ComboBox 1</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="cbComboBox1"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="lblComboBox2">
          <property name="text">
           <string>ComboBox 2</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="cbComboBox2"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ileValue3 Idx</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="RB_IdxLineEdit" name="ileValue3">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="lblSpinBox">
          <property name="text">
           <string>SpinBox</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QSpinBox" name="spSpinBox"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="lblDateTimeEdit">
          <property name="text">
           <string>DateTimeEdit</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QDateTimeEdit" name="dteDateTimeEdit">
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="lblCheckBox">
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QCheckBox" name="cbCheckBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Valid if checked</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>SelectRel</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbTest">
            <property name="text">
             <string>&amp;Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbPrevious">
            <property name="text">
             <string>&amp;Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbNext">
            <property name="text">
             <string>&amp;Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="12" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="leSelectRel"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbSelectRel">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="14" column="0" colspan="2">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblValue3">
          <property name="text">
           <string>uleValue3 Eng</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="RB_UomLineEdit" name="uleValue3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parent Table View</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="RB_TableView" name="tvParent"/>
      </item>
      <item row="0" column="0">
       <widget class="RB_ToolButtonBar" name="tbbParent" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Child Table View</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="RB_TableView" name="tvChild"/>
      </item>
      <item row="0" column="0">
       <widget class="RB_ToolButtonBar" name="tbbChild" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RB_ToolButtonBar</class>
   <extends>QWidget</extends>
   <header>rb_toolbuttonbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RB_TableView</class>
   <extends>QTableView</extends>
   <header>rb_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>RB_IdxLineEdit</class>
   <extends>QLineEdit</extends>
   <header>rb_idxlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>RB_UomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>rb_uomlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
